3.1 Harvest Visualization and Patching Documentation

obrienben opened the pull request:
Changes
- 3.1 upgrade guide
- Removed references to H1 and bandwidth
- Two new sections in the User Manual
  - User Manual -> Target Instance Quality Review -> Harvest Analysis
  - User Manual -> Target Instance Quality Review -> Patching
- Removed redundant tree view section

To build the docs, instructions are in the webcurator-docs readme

hannakoppelaar commented on webcurator-docs/guides/user-manual.rst:
```diff
@@ -1982,93 +1974,540 @@ Target Instance view page corresponding to that particular harvest which
+  - *Option* - Patching action (PRUNE or RECRAWL)
+  - *Target* - URL to be patched
+  - *File* - local file to be imported
+  - *Modified Mode* - //TODO
```
Same TODO as before

obrienben replied:
Added Modified Mode description

hannakoppelaar reviewed (commented):
There's one TODO issue concerning the semantics of the ModifiedMode field in the bulk import spreadsheet (user-manual.rst)

obrienben commented on webcurator-docs/guides/user-manual.rst:
```diff
@@ -1982,93 +1974,540 @@ Target Instance view page corresponding to that particular harvest which
 
--  **Total Failed** - The number of documents unsuccessfully downloaded
-   from "within" this site or folder.
+  - **ModifiedMode** - specify the //TODO
```
This block was actually redundant so I've removed

obrienben commented:
@hannakoppelaar have fixed the TODOs

hannakoppelaar commented:
@leefrank9527 if you're happy with it, I will merge this PR

leefrank9527 commented:
@hannakoppelaar Hi Hanna, I'm reading and testing it. Will finish that soon.

leefrank9527 commented:
@obrienben Got some comments for the doc:
1. The description below "Figure 27. Linked domain nodes": Star nodes are a single domain of URLs that also contain the primary seed of the crawl. In fact, both primary seed contained nodes and secondary seed contained nodes will be drew as star bodes.
2. Can add some description about the right click menu of the domain analysis table below "Figure 30. Domain Analysis", similar to "Figure 29. Node right-click menu".
3. Above "Figure 36. Filtering and search options", the filtering is available with the Path View as well. But it seems there is a performance problem for a big Path View for a broad filter.
4. "To reset the view and clear any fitlered or search results, click the reset button image108."" The icon image is not the latest one.
5. "Figure 52. A running patch in harvest results", "Figure 53. Patch Summary screen": Are they located in the opposite places?

obrienben commented:
Thanks @leefrank9527. I have made the relevant updates:

1. No change - there is already further description below Figure 27 detailing the star node and secondary seeds.
2. Updated to mention right-click menu for domain analysis table
3. Updated to mention that filtering is available in Crawler Path view also
4. Updated screenshots with latest icon
5. No change - to me the current descriptions match the screenshots

@hannakoppelaar  ready for you to merge
